src/editor/Editor.js

```diff
@@ -1155,13 +1155,15 @@ define(function (require, exports, module) {
      *      the "ch" parameter.
      */
     Editor.prototype.setCursorPos = function (line, ch, center, expandTabs) {
+        var oldCursorPos = this.getCursorPos(true, "start");
```

Comment:
@nethip Thus PR was actually taking care of move to next and previous context as well across files and within same files. I had de-scoped  it to only recent files as the other one needs additional UI. But these changes are generic and will be usefull for every one who wants to capture cursor jump.